src/client/visualizations/line-chart/base-chart/scale.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017-2018 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as d3 from "d3";
+
+export default function getScale(extent: number[], height: number): d3.scale.Linear<number, number> {
```

Comment:
Better name to show what it does. Basically it creates linear scale from domain and available height. Minor adjustments for readability of chart.

Reply:
it could be more specific, extent == min and max as I understood

Reply:
Yes, in d3 extent (and domain) is always passed as tuple of two numbers. In js we don't have real tuples so we use two element array.